### DISCUSSION TITLE
How to enable stickycookie option

### DISCUSSION THREAD

**Discussion by RuwaniPerera** [mitmproxy/mitmproxy#5355] [Q&A]
Based on this document I m trying to enable cookies : https://docs.mitmproxy.org/stable/overview-features/#sticky-cookies
Any suggesions

**Comment by mhils** [comment]
As the page says, you need to turn on the `stickycookie` option. There are a bunch of ways to do that, see https://docs.mitmproxy.org/stable/concepts-options/. Either in a config file, interactively, or using `--set stickycookie` when starting mitmproxy.

**Reply by devcatkit** [reply]
this option grabs the set-cookie header from response and applys it as cookie header in requests sent to server correct? 

i use this option in terminal and not work like this, set-cookie is not set as cookie. what is the option flag for the config file? 

really thanks to you, big help.
